app/tests/test_routes.py

```diff
@@ -244,3 +246,121 @@ def test_route_generate_image_manifest(
 
         assert response.status_code == 200
         assert actual_manifest == expected_image_manifest
+
+    @pytest.mark.parametrize(
+        "form_data, args_data, expected_redirect_route, expected_params",
+        [
+            # standard user with both form and args data, args has precedence for overlapping keys
+            (
+                {
+                    "transferring_body_id": "form_value",
+                    "other_param": "form_other_value",
+                },
+                {
+                    "transferring_body_id": "args_value",
+                    "another_param": "args_another_value",
+                },
+                "main.search_transferring_body",
+                {
+                    "_id": "args_value",
+                    "other_param": "form_other_value",
+                    "another_param": "args_another_value",
+                },
+            ),
+            # standard user with only form data, no transferring_body_id in args
+            (
+                {"transferring_body_id": "form_value"},
+                {},
+                "main.search_transferring_body",
+                {"_id": "form_value"},
+            ),
+            # standard user with only args data, transferring_body_id present
+            (
+                {},
+                {"transferring_body_id": "args_value"},
+                "main.search_transferring_body",
+                {"_id": "args_value"},
+            ),
+        ],
+    )
+    def test_search_route_with_various_cases_standard_user(
+        app,
+        client: FlaskClient,
+        form_data,
+        args_data,
+        expected_redirect_route,
+        expected_params,
+        mock_standard_user,
+    ):
+        mock_standard_user(client)
+
+        query_string = "&".join(
+            [f"{key}={value}" for key, value in args_data.items()]
+        )
+        url = url_for("main.search") + "?" + query_string
+
+        response = client.get(url, data=form_data)
+        assert response.status_code == 302
+
+        if expected_redirect_route == "main.search_transferring_body":
```

Comment:
same here, lets remove conditionals by splitting out permutations that require conditional test logic into their own tests

Reply:
@anthonyhashemi this is just a reminant from the older test, because we've split the tests into all access and standard user the conditional shouldnt have been there at all

Reply:
ah okay, coool